lib/ansible_tower_client/inventory_source.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleTowerClient
+  class InventorySource < BaseModel
+    def self.endpoint
+      "inventory_sources".freeze
+    end
+
+    def can_update?
+      response = api.get("#{url}update").body
```

Comment:
I'll make this change.  Per the tower docs every endpoint ends with "/" - but I like seeing `related["update"]` in the code - makes it easier to read what we're calling.